tests/logging_callback_tests/test_opentelemetry_unit_tests.py

```diff
@@ -26,15 +24,14 @@ class TestOpentelemetryUnitTests(BaseLoggingCallbackTest):
     def test_parallel_tool_calls(self, mock_response_obj: ModelResponse):
         tool_calls = mock_response_obj.choices[0].message.tool_calls
         from litellm.integrations.opentelemetry import OpenTelemetry
-        from litellm.proxy._types import SpanAttributes
 
         kv_pair_dict = OpenTelemetry._tool_calls_kv_pair(tool_calls)
 
         assert kv_pair_dict == {
-            f"{SpanAttributes.LLM_COMPLETIONS}.0.function_call.arguments": '{"city": "New York"}',
-            f"{SpanAttributes.LLM_COMPLETIONS}.0.function_call.name": "get_weather",
-            f"{SpanAttributes.LLM_COMPLETIONS}.1.function_call.arguments": '{"city": "New York"}',
-            f"{SpanAttributes.LLM_COMPLETIONS}.1.function_call.name": "get_news",
+            "gen_ai.completion.0.function_call.arguments": '{"city": "New York"}',
```

Comment:
using an fstring was just reproducing the original issue, so i'm hardcoding in what the actual value is meant to be